portfolio/target/appengine-staging/WEB-INF/appengine-generated/app.yaml

```diff
@@ -0,0 +1,45 @@
+runtime: java8
+inbound_services:
```

Comment:
I'm pretty sure we don't need to check in the target folder since this is generated by maven. Just create a file called .gitignore at the root of your project and then delete the target folder and update the PR.

https://git-scm.com/docs/gitignore

We probably need to set up a .gitignore file. Some good files to add
```
target/
pom.xml.tag
pom.xml.releaseBackup
pom.xml.versionsBackup
pom.xml.next
release.properties
dependency-reduced-pom.xml
buildNumber.properties
.mvn/timing.properties
# https://github.com/takari/maven-wrapper#usage-without-binary-jar
.mvn/wrapper/maven-wrapper.jar
```